### PR TITLE
[OpenVINO-EP] Update Documentation w.r.t latest 2021.4 Release

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description**: Update Documentation w.r.t latest 2021.4 Release

**Motivation and Context**
- Added/Updated model coverage
- New operator support
- updated ov version from 2021.3 to 2021.4
- updated the links to OpenVINO official release documentation pages
- Mentioned about the INT8 support for OV-EP

### CODE REVIEW

**Comment by MaajidKhan** [comment]
The corresponding code PR for this Release can be tracked here.
https://github.com/microsoft/onnxruntime/pull/8369

**Comment by jywu-msft** [comment]
We'll merge this one after we review and merge the 2021.4 PR

**Comment by oliviajain** [comment]
**Changes Staged**
https://oliviajain.github.io/onnxruntime/how-to/build/eps.html
https://oliviajain.github.io/onnxruntime/reference/execution-providers/OpenVINO-ExecutionProvider.html
https://oliviajain.github.io/onnxruntime/tutorials/tutorials/OpenVINO_EP_samples/squeezenet_classification_cpp.html

**Review by faxu — APPROVED** [review]

**Pull request merged**
